src/extension.ts

```diff
@@ -32,6 +34,28 @@ import AccountCommands = require('./commands/AccountCommands');
 import EnvironmentCommands = require('./commands/EnvironmentCommands');
 import PublicCommands = require('./commands/PublicCommands');
 
+/**
+ * APP INSIGHTS
+ */
+const azureInstrumentationKey = 'f9b9d6d5-2058-46ae-9ba8-56bc993474e3';
+appInsights
+	.setup(process.env.APPLICATIONINSIGHTSKEY || azureInstrumentationKey)
```

Comment:
`process.env.APPLICATIONINSIGHTSKEY` looks like never used. Candidate for removal from code?